balancer-js/src/modules/data/pool/subgraph/subgraph.ts

```diff
@@ -126,9 +118,9 @@ export class PoolsSubgraphRepository
 
     const formattedQuery = new GraphQLArgsBuilder(this.query.args).format(
       new SubgraphArgsFormatter()
-    ) as PoolsQueryVariables;
+    ) as PoolQueryVariables;
 
-    const { pools } = await this.client.Pools(formattedQuery);
+    const pools = await this.subgraphHelper.allPools(formattedQuery);
```

Comment:
This was previously only fetching first 1000 pools. Now it will fetch all.

Reply:
not sure if it's relevant, but all the pools where already fetched in the repository in `fetchDefault` functions. it's used by default when not using `fetch` which allows for a custom query.

only thing i can think of comparing allPools vs Pools is performance, so if there is no drop, let's just use allPools. At some point we are going to hit a limit anyways, but works for now.